submissions/VikTorNik/html-css-popup/index.html

```diff
@@ -0,0 +1,152 @@
+                                    <a class="content__application" href="#">
+                                        <img src="img/mail.png" alt="Mail Application">
+                                        <p>Mail</p>
+                                    </a>
+                                </li>
+                                <li class="content-button-add">
+                                    <a class="content__application" href="#">
+                                        <img src="img/maps.png" alt="Map search">
+                                        <p>Maps</p>
+                                    </a>
+                                </li>
+                                <li class="content-button-add">
+                                    <a class="content__application" href="#">
+                                        <img src="img/mars.png" alt="Mars Application">
+                                        <p>Mars</p>
+                                    </a>
+                                </li>
+                                <li class="content-button-add">
+                                    <a class="content__application" href="#">
+                                        <img src="img/news.png" alt="News reading">
+                                        <p>News</p>
+                                    </a>
+                                </li>
+                                <li class="content-button-add">
+                                    <a class="content__application" href="#">
+                                        <img src="img/photos.png" alt="Photo gallery">
+                                        <p>Photos</p>
+                                    </a>
+                                </li>
+                                <li class="content-button-add">
+                                    <a class="content__application" href="#">
+                                        <img src="img/promo.png" alt="New promotions">
+                                        <p>Promo</p>
+                                    </a>
+                                </li>
+                            </ul>
+                        </div>
+                    </li>
+                </ul>
+                </li>
+                </ul>
+            </nav>
+        </header>
+
+        <main class="main">
+            <div>
+                <a href="https://github.com/kottans" tabindex="-1" target="_blank">
+                    <img class="main__animated" width="250" src="img/logo.png" alt="Best Frontend Learning School">
+                </a>
+            </div>
+        </main>
+
+        <footer class="footer">
+            <div class="footer__div">
+                <p>&copy; 2022 • HTML і CSS практика: Hooli-style Popup by <a class="footer__git" tabindex="-1"
+                        href="https://github.com/VikTorNik" target="_blank">VikTorNik</a> </p>
+            </div>
+        </footer>
+    </div>
+</body>
+
+</html>
```

Comment:
Put an empty line at the end of every file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1). Tune your code editor's settings so it does this for you.

Reply:
Fixed it.